app/models/proposal.rb

```diff
@@ -173,6 +173,11 @@ def subscribers
     results.compact.uniq
   end
 
+  def subscribers_except_future_step_users
+    results = currently_awaiting_step_users + individual_steps.completed.map(&:user) + observers + [requester]
```

Comment:
@nickbristow if you look at the `listeners` method we deleted / rewrote, the logic in there grabs the "step users", which are in some cases shared mailboxes. Like peter said, if we want to email people, we need to look at "step completers", too. Moving in that direction may make sense, but that would be a change from what we have currently (and, like I said above, I am pretty sure that we never email completers atm....but perhaps that should change)